data/scripts/aggregate_data.py

```diff
@@ -65,8 +65,29 @@
                 senate_districts[s]["date_assumed_office"] = di['Date assumed office']
         senatefile.seek(0)
 
+common_field_names = [ "dg_small_kw",
```

Comment:
A comment might be helpful here so folks can better understand your data process

Reply:
added some comments throughout